src/fetching/putUndoEventResponse.ts

```diff
@@ -0,0 +1,27 @@
+import apiUrl from '../utils/apiUrl';
+
+import { ZetkinMembership } from '../types/zetkin';
+
+export default function putUndoEventResponse(event : number, org : number | undefined) : Promise<ZetkinMembership> {
```

Comment:
Why the return type? Looking at the body of this function, the promise it returns doesn't seem to resolve to a `ZetkinMembership`.

This same issue exists for the `putEventResponse()` function (but I can't comment twice on the same line).